### ISSUE TITLE
Resources Track is getting applications

### ISSUE THREAD

**Issue by MichaelKohler** [mozilla/Reps#168]
## Goal: 
The Resources track is open for applications.

## Related Issues & Links: 
blocked by: #113 

## Roles:

Responsible: @MichaelKohler
Accountable: Reps Council
Supporting: @couci 

## Required: 
- [x] Make sure we finish #113 (@MichaelKohler, Due: May 15th) 
- [x] Finish draft of [Resources wiki page](https://wiki.mozilla.org/ReMo/Resources) (@MichaelKohler, Due: May 17th) 
- [x] Come up with application process (where do Reps apply for it?) (@MichaelKohler, Due: May 22nd)
- [x] Write communication for all Reps to announce Resources track (@MichaelKohler, Due: May 23rd)
- [x] Send out communication through all channels (@couci, Due: May 24th)

To keep in mind: once we are using the Resources track, we need to clean up the wiki according to https://wiki.mozilla.org/ReMo/Cleanup .

**Comment by MichaelKohler** [comment]
We will test the training material with a few Reps first (direct reachout to selected active budget submitters in the past 6 months) before we open the track up for everyone. This makes sure that we have a good baseline of the training material.

**Comment by couci** [comment]
The blocker now is to get the few Reps we need to train. I will reach out to the Review Team for recommendations.

**Comment by couci** [comment]
Review team will provide a least until the 27th

**Comment by MichaelKohler** [comment]
@couci is there an update on this so we can test it and move forward?

**Comment by couci** [comment]
yes we have a list of 8 people should we minimize it or the number of 8 is ok? (btw 8 are nominated not sure if all of them will be up for it)

**Comment by MichaelKohler** [comment]
Sounds good, let's go with those.

However, not sure why you closed this issue. Reopening.

**Comment by couci** [comment]
Sorry about that. Here is the timeline I am proposing:
- [x] first review of the initial content (@kelimuttu Due: July 14th)
- [x] Draft for comms to the 8 nominees (@couci Due: July 12th)
- [x] Update training timeline on wiki (@couci, @kelimuttu Due: July 14th)
- [x] Nominees are getting communicated (@couci, @Flore-mozfr Due: July 14th)
- [x] Nominees have replied (Due: July 18th)
- [x] Training starts (Due: July 27th)

If everyone agrees I will take next steps with @kelimuttu and @Flore-mozfr

**Comment by MichaelKohler** [comment]
Quick clarification question, we had two different options last we taked about it. Is this the "evaluation of training content only" variant or the "this is an official training for feedback and these Reps will be officially part of the Resources track" variant?

Regarding timeline, I completely agree.

**Comment by couci** [comment]
I am open with both based on the nominations we have, let's bring that to the council meeting

**Comment by couci** [comment]
I've added a small section under Budget Training in [here](https://docs.google.com/document/d/1ZbS7L1PhduENa_CgNRv7zmzFC3hPKJhYu8uh_W8tKQI/edit#)
@Flore-mozfr @kelimuttu please review so I can upload the wiki

**Comment by couci** [comment]
Doc for comms is [here](https://docs.google.com/document/d/1l4ZHM8TzK6fRga311kSXs0VcdiZgn9TqrXn-G5F4uIE/edit) Currently only view mode
@Flore-mozfr @kelimuttu please review

**Comment by couci** [comment]
I wrote about [the workflow and the comms that will go out during the training](https://docs.google.com/document/d/189ppyoTHXqnJeSG0aJZf4N_v10TfVs75eKwHEWY-lIM/edit#heading=h.vzh2lnc3rpx1)
Please review

**Comment by couci** [comment]
Email sent to potential testers

**Comment by kelimuttu** [comment]
Put some minor suggestions on the doc. Thanks, K

**Comment by couci** [comment]
wiki has been updated

**Comment by kelimuttu** [comment]
As I tried Teachable system last week, there's no way for the teacher to see how much the participants scored on the quiz. So I think we need to ask the participant to screenshot their result and bring it on the weekly meeting.

**Comment by couci** [comment]
Thanks @kelimuttu I will change our comms emails accordingly. Let's wait about people replies until the 18th, to move this forward

**Comment by kelimuttu** [comment]
Action items for the track opening: 
- [x] Comms for the opening (e-mail and Discourse)
- [x] Application form

**Comment by MichaelKohler** [comment]
@couci @kelimuttu I think we can close this issue as the Resources track is open and create a new one for the remaining and upcoming work. What do you think?

**Comment by kelimuttu** [comment]
You're right. I've made another issue for the ongoing training for India Reps [here](https://github.com/mozilla/Reps/issues/252).
